addons/stock/models/stock_picking.py

```diff
@@ -473,7 +473,7 @@ def _compute_move_line_exist(self):
 
     def _compute_has_packages(self):
         for picking in self:
-            picking.has_packages = picking.move_line_ids.filtered(lambda ml: ml.result_package_id)
+            picking.has_packages = bool(self.env['stock.move.line'].search_count([('picking_id', '=', picking.id), ('result_package_id', '!=', False)]))
```

Comment:
Maybe add `limit=1` or use `read_group`

Reply:
A `limit` for a `search_count` is useless.
We can make a read_group but in this case the has_package is only use in the form. I miss something ?

Reply:
OK, no problem